app/app.html

```diff
@@ -14,7 +17,6 @@
         }
       }());
     </script>
-    <script type="text/javascript" src="http://media.twiliocdn.com/sdk/js/client/v1.3/twilio.min.js"></script>
```

Comment:
will this break twilio functionality? I saw twilio.js was added to the package.json does that resolve this? When I tried using the npm package I was unable to get working. Were you able to get that working?